xml_converter/integration_tests/test_cases/achievement_bitmask_valid/input/pack/xml_file.xml

```diff
@@ -7,13 +7,6 @@
 		<POI AchievementBit="5" Type="mycategory" XPos="169.81" YPos="210.65" ZPos="215.83" MapID="50" />
 		<POI AchievementBit="2147483647" Type="mycategory" XPos="169.81" YPos="210.65" ZPos="215.83" MapID="50" />
 		<Trail AchievementBit="5" Type="mycategory" />
-
-		<!-- TODO: textproto seems to think a fixed32 is a uint32, but we support negative values for this currently. We should probably adjust that -->
-		<POI AchievementBit="-2147483648" Type="mycategory" XPos="169.81" YPos="210.65" ZPos="215.83" MapID="50" />
-		<POI AchievementBit="-5" Type="mycategory" XPos="169.81" YPos="210.65" ZPos="215.83" MapID="50" />
-
-		<!-- TODO: These throw errors and crash but probably should not if this were a uint32 -->
-		<!-- <POI AchievementBit="4294967295" Type="mycategory" XPos="169.81" YPos="210.65" ZPos="215.83" MapID="50" /> -->
-		<!-- <POI AchievementBit="2147483648" Type="mycategory" XPos="169.81" YPos="210.65" ZPos="215.83" MapID="50" /> -->
+		<POI AchievementBit="2147483647" Type="mycategory" XPos="169.81" YPos="210.65" ZPos="215.83" MapID="50" />
```

Comment:
This line is identical to line 8 now. Is this intended?

Reply:
Good catch, removed.